src/home/WalletHome.tsx

```diff
@@ -56,6 +58,16 @@ function WalletHome() {
   const showNotificationCenter = getFeatureGate(StatsigFeatureGates.SHOW_NOTIFICATION_CENTER)
   const showNotificationSpotlight = showNotificationCenter && canShowNotificationSpotlight
 
+  const [hideBalances, setHideBalances] = useState(false)
+  const eyeIconOnPress = () => {
```

Comment:
can we get some test coverage for this? Should be fine to do some test with the HomeTokenBalance and TxFeed components to ensure balances are hidden / shown when the icon is clicked.